src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -7,132 +7,157 @@ import {t, tct} from 'app/locale';
-                <ReasonItem key={i}>
-                  <OwnershipTag tagType={type} />
-                  <OwnershipValue>{matched}</OwnershipValue>
-                </ReasonItem>
-              ))}
-            </div>
+            <HovercardHeader>
+              <HovercardActorAvatar actor={actor} />
+              {actor.name || actor.email}
+            </HovercardHeader>
+            {actor.id === undefined && (
+              <EmailAlert icon="icon-warning-sm" type="warning">
+                {tct(
+                  'The email [actorEmail] is not a member of your organization. [inviteUser:Invite] them or link additional emails in [accountSettings:account settings].',
+                  {
+                    actorEmail: <strong>{actor.email}</strong>,
+                    accountSettings: <Link to="/settings/account/emails/" />,
+                    inviteUser: (
+                      <a
+                        onClick={() =>
+                          openInviteMembersModal({
+                            initialData: [
+                              {
+                                emails: new Set([actor.email]),
+                              },
+                            ],
+                            source: 'suggested_assignees',
+                          })
+                        }
+                      />
+                    ),
+                  }
+                )}
+              </EmailAlert>
+            )}
           </React.Fragment>
-        )}
-      </HovercardBody>
-    }
-    {...props}
-  />
-);
-
-SuggestedOwnerHovercard.propTypes = {
-  /**
-   * The suggested actor.
-   */
-  actor: PropTypes.oneOfType([
-    // eslint-disable-next-line react/no-typos
-    SentryTypes.User,
-    // Sentry user which has *not* been expanded
-    PropTypes.shape({
-      email: PropTypes.string.isRequired,
-      id: PropTypes.string.isRequired,
-      name: PropTypes.string.isRequired,
-    }),
-    // Unidentifier user (from commits)
-    PropTypes.shape({
-      email: PropTypes.string.isRequired,
-      name: PropTypes.string.isRequired,
-    }),
-    // Sentry team
-    PropTypes.shape({
-      id: PropTypes.string.isRequired,
-      name: PropTypes.string.isRequired,
-    }),
-  ]),
-
-  /**
-   * The list of commits the actor is suggested for. May be left blank if the
-   * actor is not suggested for commits.
-   */
-  commits: PropTypes.arrayOf(
-    PropTypes.shape({
-      message: PropTypes.string.isRequired,
-      dateCreated: PropTypes.string.isRequired,
-    })
-  ),
-
-  /**
-   * The list of ownership rules the actor is suggested for. Maybe left blank
-   * if the actor is not suggested based on ownership rules.
-   */
-  rules: PropTypes.arrayOf(PropTypes.array),
-};
+        }
+        body={
+          <HovercardBody>
+            {commits !== undefined && (
+              <React.Fragment>
+                <div className="divider">
+                  <h6>{t('Suspect Commits')}</h6>
```

Comment:
Are we sure we want to change this? `Suspect Commits` is a slightly different feature

Reply:
That was what Adi suggested, we can revert it

Reply:
hmm really? I see that I just mentioned changing "ownership rules" to "matching ownership rules"

Reply:
I don't know why I read it as that, I will change the titles to `Commits` and `Matching Ownership Rules`